mc-sema/cfgToLLVM/RegisterUsage.cpp

```diff
@@ -108,12 +110,25 @@ std::map<MCSemaRegs, RegInfo> REG_TO_OFFSET_MAP {
 };
 
 StringRef getRegisterName(MCSemaRegs reg) {
-   return REG_TO_OFFSET_MAP.at(reg).name;
+    try {
+        return REG_TO_OFFSET_MAP.at(reg).name;
+    } catch (const std::out_of_range &oor) {
+        std::cerr << __FILE__ << ":" << __LINE__ << ": Could not find register name for: " << reg << std::endl;
+        std::cerr <<  oor.what() << std::endl;
+        throw;
+    }
 }
 
 int getRegisterOffset(MCSemaRegs reg) {
-	return REG_TO_OFFSET_MAP.at(reg).position;
+    try {
+        return REG_TO_OFFSET_MAP.at(reg).position;
+    } catch (const std::out_of_range &oor) {
```

Comment:
In what cases is this exception hit? If you hit this exception, do you modify the state structure, or file a bug?